packages/cli/src/utils/formatting.js

```diff
@@ -36,3 +32,32 @@ export function colorize(text) {
     .replace(/ (-[-\w,]+)/g, m => chalk.bold(m))
     .replace(/`(.+)`/g, (_, m) => chalk.bold.cyan(m))
 }
+
+export function box(message, title, options) {
+  return boxen([
+    title || chalk.white('Nuxt Message'),
+    '',
+    chalk.white(foldLines(message, 0, 0, maxCharsPerLine()))
+  ].join('\n'), Object.assign({
+    borderColor: 'white',
+    borderStyle: 'round',
+    padding: 1,
+    margin: 1
+  }, options)) + '\n'
+}
+
+export function warningBox(message, title) {
+  return box(message, title || chalk.yellow('❗ Nuxt Warning'), {
```

Comment:
What about ⚠️ ? :D

Reply:
Yeah, I should've known to use that one as consola also uses it. Actually now I searched for the ✖ char, noticed it was a 'heavy multiplication sign' unicode char and then searched for 'heavy exclamation mark' ;)

Reply:
Yes. Would be a better idea to keep consistency (Emoji or ascii anywhere) Also windows support should be considered.